Library/Formula/pure.rb

```diff
@@ -0,0 +1,23 @@
+class Pure < Formula
+  desc "functional programming language based on term rewriting"
+  homepage "http://purelang.bitbucket.org"
+  url "https://bitbucket.org/purelang/pure-lang/downloads/pure-0.64.tar.gz"
+  sha256 "efd55229342aec9d79e8fa4732a30f140e1773064f3869abde053e01468f7b07"
+
+  depends_on "llvm34"
```

Comment:
Formulae in core aren't allowed to depend on taps; this is why Pure was removed in https://github.com/Homebrew/homebrew/pull/39218.